### PR TITLE
Enable core uploader service

### PR DESCRIPTION
**Pull request by renukamanavalan**
If core-storage secret key is available, add to /etc/sonic/core_analyzer.rc.json and enable & start core_uploader service

If https_proxy is provided, update /etc/sonic/core_analyzer.rc.json.

### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [] Bug fix
- [] Testbed and Framework(new/improvement)
- [] Test case(new/improvement)

### CODE REVIEW

File: ansible/config_sonic_basedon_testbed.yml [hunk 1]
```diff
@@ -144,6 +144,41 @@
         become: true
         when: stat_result.stat.exists is defined and stat_result.stat.exists
 
+      - name: read account key
+        set_fact:
+            core_key: "{{ corefile_uploader['azure_sonic_core_storage']['account_key'] | default('') }}"
```
Comment (lguohan) [inline ansible/config_sonic_basedon_testbed.yml]: what if corefile_uploader is not defined? will this fail the job?

Comment (renukamanavalan) [reply ansible/config_sonic_basedon_testbed.yml]: Yep it fails the job. It fails for any missing field in the path, except for last key, which is saved by default. Thanks for catching. I only tested for incorrect value for last key in the path.

File: ansible/config_sonic_basedon_testbed.yml [hunk 2]
```diff
@@ -144,6 +144,41 @@
...
+            regexp: '^.*account_key'
+            line: '        "account_key": "{{ core_key }}",'
+        become: true
+        when: core_key != ""
+
+      - name: Put https-proxy in core_analyzer.rc.json
+        lineinfile:
+            name: /etc/sonic/core_analyzer.rc.json
+            regexp: '^.*https_proxy'
+            line: '        "https_proxy": "{{ corefile_uploader['env']['https_proxy'] }}"'
```
[the remark this answers is not in the document]
Comment (renukamanavalan) [reply ansible/config_sonic_basedon_testbed.yml]: Thanks. Done.

File: ansible/config_sonic_basedon_testbed.yml [hunk 3]
```diff
@@ -144,6 +144,41 @@
...
+      - name: read account key
+        set_fact:
+            core_key: "{{ corefile_uploader['azure_sonic_core_storage']['account_key'] }}"
+        when: corefile_uploader['azure_sonic_core_storage']['account_key'] is defined
+
+      - name: Put secret in core_analyzer.rc.json
+        lineinfile:
+            name: /etc/sonic/core_analyzer.rc.json
+            regexp: '^.*account_key'
+            line: '        "account_key": "{{ core_key }}",'
```
[the remark this answers is not in the document]
Comment (renukamanavalan) [reply ansible/config_sonic_basedon_testbed.yml]: Thanks. Done.

**Comment by lguohan** [comment]
broken pr test. revert.

https://sonic-jenkins.westus2.cloudapp.azure.com/job/vs/job/buildimage-vs-image-pr/3108/console

**Pull request merged**